app/views/content/support-for-early-career-teachers.md

```diff
@@ -55,7 +55,9 @@ This is in addition to the time allocated for planning, preparation and assessme
 
 ## Meeting the teachers’ standards
 
-You’ll have to show you meet the [teachers’ standards](https://www.gov.uk/government/publications/teachers-standards) through your induction period. You’ll need to pass your induction period to continue to work as a qualified teacher in a state maintained school in England.
+You’ll have to show you meet the [teachers’ standards](https://www.gov.uk/government/publications/teachers-standards) at the end of your induction period — your mentor and induction tutor will help you to develop your skills and practice to support you with this.
```

Comment:
I did want this to be a bit more reassuring for users, and talk about how likely it was that an ECT would pass/fail their induction.

Had a bit of back and forth with policy about this, but they were keen that we still emphasised that it is important for ECTs to pass their induction and they didn't want to devalue the assessment element.

This is what we settled on! @gemmadallmandfe @philspray @KeishaDfE